Yoast/wordpress-seo
# Stories/7912 change feed url

## Summary

This PR can be summarized in the following changelog entry:

No functional changes.

## Relevant technical choices:

* Changes the feed URL to `https://yoa.st/widget-feed` so that it can be redirected later without needing any code changes.

## Test instructions

This PR can be tested by following these steps:

* Build the JS files.
* Go to the dashboard.
* You should see the two latest yoast.com posts in the Dashboard Widget.

Fixes #7912.
